index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
```

Comment:
I suggest adding the language property. browsers use the lang property to identify how the document will be read.